security/PracticalSecurity.md

```diff
@@ -0,0 +1,11 @@
+On the last day of work that I did in November, the Friday before I was set to start Lambda school, my cell phone was stolen. Stolen, actually, might be a strong word for what happened. I was riding my bike to work and my phone fell out of my pocket and I didn't realize it until I got to work. 
+
+I didn't have a lock screen on my phone. 'I always keep my phone on me,' I said at the time, 'so I'm not too worried'. I probably should have thought that through a bit deeper. We don't need security for when things are going well. We need security for the times in which they are not going well. 
+
+My phone had my passwords saved, my credit card saved, which is very convenient for when I want to make a quick purchase. It is much less convenient when my phone is unlocked and in the hands of a stranger.
+
+I am very lucky that Google thinks that security is important, even when I don't. My phone was a Google phone and I was able to log on to my Google account from my work computer, lock my phone and send an alert to the screen, asking it to be returned to me. I was able to track my phone's movement across the city and repeatedly call it, overriding my settings that the ringer was on silent--to no avail. It became clear the person had no intention of getting my phone back to me. I was able to remotely wipe the device of all my content, even with the phone clear across town.
+
+I say this now because, five months later, I should know better. But it wasn't until today's assignment that I actually put a screen lock on my current phone. 
```

Comment:
unreal story!!! ugh android lol jk but sigh of relief you got that lock on there